app/views/users/profile.html.erb

```diff
@@ -195,6 +189,15 @@
     <div class="tab-pane" id="comments">
       <h2 style="text-align:center;"><i class="fa fa-spinner fa fa-spin"></i></h2>
     </div>
+
+    <div class="tab-pane" id="barnstars">
+      <% if @user.user %>
+      <% @user.user.barnstars.each do |tag| %>
+        <p style="color:#aaa;"><i style="color:#db4;" class="fa fa-large fa fa-star-o"></i> <i><%= raw t('users.profile.barnstar_awarded_to_by', :url1 => "/profile/"+tag.node.author.name, :author => tag.node.author.name, :url2 => "/wiki/barnstars#"+tag.name.split('-').each{|w| w.capitalize!}.join('+'), :barnstar => tag.name.split(':').last.split('-').each{|w| w.capitalize!}.join(' '), :url3 => "/profile/"+tag.author.username, :awarder => tag.author.username, :url4 => tag.node.path, :work => tag.node.title) %></i></p>
```

Comment:
There is a little bug out there that displays the incorrect url. `:url2` will be `"/wiki/barnstars#"+tag.name.split(':').last.split('-').each{|w| w.capitalize!}.join('+')`